openshift-container-platform-4/policies/CM-Configuration_Management/component.yaml

```diff
@@ -650,26 +650,49 @@
 - control_key: CM-7 (5)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
   narrative:
     - key: a
       text: |
-        A complete control response is planned. Engineering progress can be
-        tracked via:
+        Red Hat OpenShift allows for configuring the platform in such a way
+        that only signed containers will be accepted for scheduling. [1]
 
-        https://issues.redhat.com/browse/CMP-205
+        Red Hat signs the images provided for running the platform, as well
+        as the base images that are usable for building other workloads.
+
+        Customers should be careful to enforce image signing on their own images.
+
+        It is relevant to configure the policy file to only accept containers
+        from authorized registries.
```

Comment:
This reply is good and should be included but I wonder if the control response should also mention container registry whitelist?

Reply:
I thought that was implied from the response. Could you help me rephrase this?

Reply:
Ah, sorry, I read the sentence as "only accept containers from signed registries" for some reason. Sorry.
But maybe including this link would be better: https://docs.openshift.com/container-platform/4.7/security/container_security/security-deploy.html#security-deploy-image-sources_security-deploy
because here the default policy is "reject" and that's what I assumed the CaC rule for this control would be checking.